tests/test_pluggable.py

```diff
@@ -334,6 +431,38 @@ def test_retrieve_subject_token_saml(self):
 
             assert subject_token == self.EXECUTABLE_SAML_TOKEN
 
+    @mock.patch.dict(os.environ, {"GOOGLE_EXTERNAL_ACCOUNT_ALLOW_EXECUTABLES": "1"})
```

Comment:
Might also be worth it to add a test that tests changing between interactive/non-interactive modes, but not sure what you're able to validate given the way the code is written

Reply:
Doing some refactoring